src/main/java/TaskManager.java

```diff
@@ -0,0 +1,111 @@
+import java.io.IOException;
+import java.util.ArrayList;
+public class TaskManager {
+    private ArrayList<Task> todoList;
+
+    public TaskManager(){
+        this.todoList=new ArrayList<>();
+    }
+
+    protected Task addNewTask(String[] taskInfo) throws DukeException{
+        if(taskInfo[0].equals("todo")){
+            return this.addTodo(taskInfo[1]);
+        }else if(taskInfo[0].equals("event")){
+            return this.addEvent(taskInfo[1]);
+        }else if (taskInfo[0].equals("deadline")){
+            return this.addDeadline(taskInfo[1]);
+        }
+        throw new DukeException("Something wrong lol");
```

Comment:
Haha. I like the humour (:

Reply:
hi! this was added in frustration because this, by right, should never be thrown if the Parser class works as intended 😒